### ISSUE TITLE
Update to latest version

### ISSUE THREAD

**Issue by jomo** [xmikos/fdroiddata#44]
Got that expiry warning again, would be great if you could update it before it expires (next month).
Thanks for your effort!

**Comment by DrDustwolf** [comment]
Copy that. Thank you in advance.

**Comment by DrDustwolf** [comment]
Do you work on an update or do  have you abandoned libresignal?

**Comment by nehrwert** [comment]
yes, if you would push another update it would be great, thanks for your work!

**Comment by mimi89999** [comment]
I really doubt there will be any new version.

**Comment by docloy** [comment]
Would be awesome if there would be an update.... 

I'd love not to use it. But a lot of people do use it to text with me. And they wouldn't be able to use conversations.... :-/

Thanks in advance!

**Comment by sheldonelectric** [comment]
There's a new version 4.12.3 but it's not in the repository.
Signal 4.9.6 has expired and therefore doesn't work at all.

**Comment by jomo** [comment]
I ended up switching to the offical build:

0. Make a backup of LibreSignal *data* using [oandbackup](https://f-droid.org/packages/dk.jens.backup/)
0. Uninstall LibreSignal
0. Download and install the official app from [here](https://signal.org/android/apk/)
  (don't run it yet)
0. Restore "LibreSignal" *data* from oandbackup.
0. Run Signal as usual

That way all your conversations and your encryption keys are preserved.

**Comment by jomo** [comment]
If I find the time to do it, I'll set up an F-Droid repository that simply provides the [latest apk](https://updates.signal.org/android/latest.json) for the sake of easy installation and updates through F-Droid.

Last time I tried doing that it failed because of [fdroid/fdroidserver#229](https://gitlab.com/fdroid/fdroidserver/issues/229). Perhaps someone with more time on their hands finds a way to do fix this.
